notes/coredns-1.9.0.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "CoreDNS-1.9.0 Release"
+description = "CoreDNS-1.9.0 Release Notes."
+tags = ["Release", "1.9.0", "Notes"]
+release = "1.9.0"
+date = "2022-02-01T00:00:00+00:00"
+author = "coredns"
++++
+
+This is a release with bug fixes and some new features added. Starting with 1.9.0
+the minimal required go version will be 1.17.
+Wildcard queries are no longer supported by the _kubernetes_ plugin.
+
+
+## Brought to You By
+
+Chris O'Haver,
+Ondřej Benkovský,
+Tomas Hulata,
+Yong Tang,
+xuweiwei
+
+## Noteworthy Changes
+
+* plugin/kubernetes: remove wildcard query functionality (https://github.com/coredns/coredns/pull/5019)
+* Health-checks should respect force_tcp (https://github.com/coredns/coredns/pull/5109)
+```suggestion
```

Comment:
@yongtang this line with suggestion is breaking up release notes rendering

Reply:
My bad.  my browser kept adding extraneous `suggestion` markdowns when using the "suggestion" feature. I thought I cleaned them all up, but missed one 

* #5175